dnf/cli/commands/history.py

```diff
@@ -344,11 +363,10 @@ def run(self):
             elif vcmd == 'userinstalled':
                 ret = self._hcmd_userinstalled()
             elif vcmd == 'store':
-                transactions = self.output.history.old(tids)
-                if not transactions:
-                    raise dnf.cli.CliError(_('Transaction ID "{id}" not found.').format(id=tids[0]))
-
-                data = serialize_transaction(transactions[0])
+                tid = self.history_get_transaction(tids)
+                if not tid:
+                    sys.exit(1)
```

Comment:
Please remove the `if` with the `sys.exit(1)`, after changing the `history_get_transaction{,s}` to raise exceptions, it should no longer even occur.